src/transformers/convert_slow_tokenizer.py

```diff
@@ -40,21 +40,24 @@ def __init__(self, model: str):
         self.sp = SentencePieceProcessor()
         self.sp.Load(model)
 
-    def extract(self) -> Tuple[Dict[str, int], List[Tuple]]:
+    def extract(self, vocab_scores=None) -> Tuple[Dict[str, int], List[Tuple]]:
```

Comment:
Maybe add a little doc on the vocab, what happens when it defaults

Reply:
It's added underneath